website/docusaurus.config.js

```diff
@@ -25,11 +25,6 @@ module.exports = {
                 src: 'img/logo.svg',
             },
             items: [
-                {
```

Comment:
why are you removing these?

Reply:
Good question! I removed this one because that link doesn't go anywhere new. Now that all docs are on the same page, it doesn't really serve a purpose anymore. The link right next to it ("Unleash") takes you to the same place. 

Furthermore, because of the highlighting regex it would sometimes be link-colored, and sometimes not, which is both inconsistent and confusing.

However, I'd be happy to leave it in if you think it still serves a purpose?

Reply:
@ivarconr Thoughts on this? If you'd prefer to leave it in, I'm fine with that, but it'll take a little more styling caution (or maybe just removing the regex?) and it may be more trouble than it's potentially worth.

Reply:
tbh I was a bit fast and though this was a redirect rule. All fine!